### PR TITLE
fix(severity): Update default rate limits

### PR DESCRIPTION
**Pull request by snigdhas**
Updates the default rate limits - the options are updated in https://github.com/getsentry/sentry-options-automator/pull/1020

### CODE REVIEW

File: src/sentry/event_manager.py [hunk 1]
```diff
@@ -146,8 +146,8 @@
 
 HIGH_SEVERITY_THRESHOLD = 0.1
 
-SEER_GLOBAL_RATE_LIMIT_DEFAULT = 20  # 20 requests per second
-SEER_PROJECT_RATE_LIMIT_DEFAULT = 5  # 5 requests per second
+SEER_GLOBAL_RATE_LIMIT_DEFAULT = 1200  # 1200 requests per minute
+SEER_PROJECT_RATE_LIMIT_DEFAULT = 300  # 300 requests per minute
```
Comment (scefali) [inline src/sentry/event_manager.py:150]: @snigdhas 300 seems rather high to me

Comment (snigdhas) [reply src/sentry/event_manager.py:150]: What would you suggest? right now, it's the same limit as before but spread out over a minute

**Review by wedamija — COMMENTED** [review]
Is the idea here to try and prevent bursts from hitting the cap?

The downside of a larger limit over a longer period of time is that now one burst can disable seer for a whole minute

**Comment by snigdhas** [comment]
yeah, that's the goal - we're hitting the [project limit](https://app.datadoghq.com/dashboard/v2m-z9p-w5g?fromUser=true&fullscreen_end_ts=1712163370318&fullscreen_paused=false&fullscreen_refresh_mode=sliding&fullscreen_section=overview&fullscreen_start_ts=1712076970318&fullscreen_widget=6703398111443222&refresh_mode=sliding&view=spans&from_ts=1712075902815&to_ts=1712162302815&live=true) a lot at only 10% rolled out. 

maybe something like 75 requests over 15 seconds?

**Comment by wedamija** [comment]
> yeah, that's the goal - we're hitting the [project limit](https://app.datadoghq.com/dashboard/v2m-z9p-w5g?fromUser=true&fullscreen_end_ts=1712163370318&fullscreen_paused=false&fullscreen_refresh_mode=sliding&fullscreen_section=overview&fullscreen_start_ts=1712076970318&fullscreen_widget=6703398111443222&refresh_mode=sliding&view=spans&from_ts=1712075902815&to_ts=1712162302815&live=true) a lot at only 10% rolled out.
> 
> maybe something like 75 requests over 15 seconds?

Realistically you're going to keep hitting the limits most likely, but bumping things here won't hurt. Maybe it'd be better to just make both the window and the limit configurable via options, then you can change them more easily?

**Review by scefali — APPROVED** [review]

**Pull request merged**